theme/snippets/image-caption-module.liquid

```diff
@@ -22,17 +27,36 @@
       col-12 sm:col-9
       {% endif %}
     " 
-    src="{{ image_module_image.src }}" 
-    alt="{{ image_module_image.alt }}
+    src="{{ image_module_first_image.src }}" 
+    alt="{{ image_module_first_image.alt }}
     "/>
     {% if caption %}
-      <span class="{{ color }} pt_25 sans-xs
+      <span class="{{ color }} pt_25 sm:pt0 sans-xs
         {% if variant == 'left' %}
-        col-12 sm:col-3 sm:pl2_5
+        col-12 sm:col-3 sm:ml2_5
         {% endif %}
         ">
         {{ caption }}
       </span>
     {% endif %}
   </div>
 {% endif %}
+
+
+{% if variant == 'two-up' and image_module_first_image and image_module_last_image %}
```

Comment:
Where is `image_module_last_image` defined?

Reply:
Oops deleted this that I had previously defined because it didn't seem necessary!